doc/browsing_cubes/intro.rst

```diff
@@ -1,9 +1,54 @@
 Browsing Cubes
 ==============
 
-This is a description of how you can use the cube_browser package to look at cubes and make plots and sliders without having to make any widgets.
+The Cube Browser package offers an API which you can use to visualize your cubes in a Jupyter notebook.
+It utilizes the Python, Matplotlib, Iris and Cartopy libraries while offering the additional functionality of plotting an entire cube without having to slice it.
 
-Links to Notebooks
-------------------
+You can make use of the Cube Browser library to write a short notebook to display your Iris cube data.
+There are several advantages of using this approach, but the main advantage is flexibility in the options you can apply to each plot.
+Additionally, if you know what plots you would like and how you would like them laid out, this is a very quick way of producing your plots and exploring your cube data.
```

Comment:
It would be useful to have another sentence at the end of this that mentions that you can then reuse the thing you've produced with tomorrow's/next week's/last century's data.